extensions/default/src/customizations/studyBrowserCustomization.ts

```diff
@@ -94,4 +94,5 @@ export default {
         },
     ],
   },
```

Comment:
Here there is the potential for code duplication. In fact, it might be best to refactor these two customization points and make the callbacks commands. Please consider this. Thanks.